src/model/SizingSystem.cpp

```diff
@@ -666,6 +693,19 @@ namespace model {
       return result;
     }
 
+    bool SizingSystem_Impl::setOccupantDiversity(boost::optional<double> occupantDiversity) {
```

Comment:
There's no point taking an optional double here, this is an antiquated way of doing things IMHO. Change to double only